.github/workflows/trigger-site-build.yaml

```
@@ -6,9 +6,8 @@ jobs:
   build:
     uses: ProjectPythia/cookbook-actions/.github/workflows/build-book.yaml@main
     with:
-      environment_file: 'environment.yml'
       environment_name: pythia
       artifact_name: book-zip-${{ github.event.number }}
       path_to_notebooks: 'portal'
-      build_command: 'make -j4 html'
+      base_url: '/${{ github.event.repository.name }}/_preview/${{ github.event.number }}'
```

I think we have to handle `base_url` slightly differently for this repo because it deploys to `projectpythia.org` without any subdirectories.

This line should probably be
```suggestion
      base_url: '/_preview/${{ github.event.number }}'
```